components/accordion/private/panel.jsx

```diff
@@ -0,0 +1,78 @@
+/* Copyright (c) 2015-present, salesforce.com, inc. All rights reserved */
+/* Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license */
+
+// Implements the [Accordion design
+// pattern](https://www.lightningdesignsystem.com/components/accordion/) in
+// React. Based on SLDS v2.3.2
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Button from '../../button';
+
+import { ACCORDION_PANEL } from '../../../utilities/constants';
+
+const propTypes = {
+	/**
+	 * The panel content for the Accordion component. Event handler for the accordion panels should be added here `<Panel onTogglePanel.../>`. Optional `panelContentActions` component may be passed as prop. _Tested with Mocha framework._
+	 */
+	children: PropTypes.node,
+	/**
+	 * Indicates whether item is expanded or not, which should be handled by `onTogglePanel`. _Tested with Mocha framework._
+	 */
+	expanded: PropTypes.bool.isRequired,
+	/**
+	 * ID of the item belonging to this panel. _Tested with snapshot testing._
+	 */
+	id: PropTypes.oneOfType([PropTypes.number, PropTypes.string]).isRequired,
+	/**
+	 * Optional component that can be passed as prop to `<Panel />`. As an example, a menu dropdown could be used here to handle additional actions for each accordion panel. _Tested with Mocha framework._
+	 */
+	panelContentActions: PropTypes.node,
+	/**
+	 * Callback that will run whenever a panel is toggled. Function should handle state to toggle `expanded` prop. _Tested with Mocha framework._
+	 */
+	onTogglePanel: PropTypes.func.isRequired,
+	/**
+	 * Summary header of the item belonging to this panel. _Tested with Mocha framework and snapshot testing._
+	 */
+	summary: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired,
+	/**
+	 * HTML title attribute. _Tested with snapshot testing._
```

Comment:
Please tell folks what element this is on--the summary? The purpose would be great, too. "The summary is truncated and this title allows the user to view the entire summary."

Reply:
Thanks for this! I tested and realized it wasn't truncating. I figured out the extra class it needed since it's using flexbox.